website/src/components/Visualisations/Achievements/AcheivementEntries.ts

```diff
@@ -153,11 +153,17 @@ const turnsInPowerMetricCollection = (data: OutputJSONType) =>
 
 const acheivementList: AcheivementEntry[] = [
```

Comment:
AchievementEntry

Reply:
oof

Reply:
should this be changed in another PR or leave it?